iml-api/src/graphql/mod.rs

```diff
@@ -566,6 +567,48 @@ impl QueryRoot {
             },
         })
     }
+
+    async fn server_profiles(context: &Context) -> juniper::FieldResult<ServerProfileResponse> {
+        let server_profile_records = sqlx::query!(
```

Comment:
```suggestion
        let server_profile_records = sqlx::query_as!(ServerProfileRecord,
```

Could we not just use query_as here?

Reply:
No, we're querying repository data via joins here as well (`jsonb_agg ... AS repos`). We get a struct with all the fields of `ServerProfileRecord` and `repos` field containing `serde_json::Value` with repositories.

Reply:
I wonder if we could cast the item here to it's final type and avoid the need to do it later